cli/constants.py

```diff
@@ -2,6 +2,8 @@
 Provide constants for command line interface.
 """
 ADDRESS_REGEXP = '[0-9a-f]{70}'
+BATCH_ID_REGEXP = '^[0-9a-f]{128}$'
+BLOCK_ID_REGEXP = '^[0-9a-f]{128}$'
```

Comment:
we have in remme-client-python this: https://github.com/Remmeauth/remme-client-python/blob/17901e34f3e46c4decd6452e675f189a4f068db1/remme/models/general/patterns.py#L13. Why we need to copy/paste to another project?

Reply:
Done.